pandas/core/generic.py

```diff
@@ -7623,12 +7624,24 @@ def between_time(
             Initial time as a time filter limit.
         end_time : datetime.time or str
             End time as a time filter limit.
-        include_start : bool, default True
+        include_start : bool
             Whether the start time needs to be included in the result.
-        include_end : bool, default True
+
+            .. deprecated:: 1.4.0
+                Arguments `include_start` and `include_end` have been deprecated
+                to standardize boundary inputs. Use `inclusive` instead, to set
+                each bound as close or open.
```

Comment:
does this render correctly? could you include a screenshot of this docstring please?

Reply:
I checked and indeed, it doesn't render correctly. However, this does pass the checks. I'm not sure what we're missing here.

Reply:
can you check other docstrings which use `.. deprecated::`? Might be the missing blankline

Reply:
```suggestion
            .. deprecated:: 1.4.0
               Arguments `include_start` and `include_end` have been deprecated
               to standardize boundary inputs. Use `inclusive` instead, to set
               each bound as close or open.
```
don't need a blank line. I never include blank line before next arg, but the correct spacing might be important?